RadixWallet/Core/DesignSystem/ViewModifiers/NavigationBarAppearenceViewModifier.swift

```diff
@@ -0,0 +1,39 @@
+import SwiftUI
+
+extension View {
+	func configureNavigationBarAppearence() -> some View {
```

Comment:
I am really not sure that we want to do UIKit magic for this, we are trying move away from non-standard UI solutions.

Reply:
I also don't love this, but SwiftUI doesn't have a way of setting a default back button for every view. Only way to do it would be to manually add it on every view, and that would also involve checking for the case we want a close button on the leading bar (and should not add the back button). 

Also, this PR moves away from some UIKit + Introspect magic so I still consider it a win in that side

![Screenshot 2024-05-17 at 15 07 30](https://github.com/radixdlt/babylon-wallet-ios/assets/164921079/69a81718-8a8c-4da7-b685-cfd75121a3d2)